cmd/fyne_demo/screens/container.go

```diff
@@ -156,3 +160,37 @@ func makeTableTab() fyne.CanvasObject {
 			label.SetText(fmt.Sprintf("Cell %d, %d", row+1, col+1))
 		})
 }
+
+func makeTreeTab() fyne.CanvasObject {
+	left := make(map[string][]string)
+	internalwidget.AddTreePath(left, "A", "B", "C", "abc")
+	internalwidget.AddTreePath(left, "A", "D", "E", "F", "adef")
+	internalwidget.AddTreePath(left, "A", "D", "E", "G", "adeg")
+	internalwidget.AddTreePath(left, "A", "H", "I", "ahi")
+	internalwidget.AddTreePath(left, "A", "J", "K", "ajk")
+	internalwidget.AddTreePath(left, "A", "L", "M", "N", "almn")
+	internalwidget.AddTreePath(left, "A", "O", "ao")
+	internalwidget.AddTreePath(left, "A", "P", "Q", "R", "apqr")
+	internalwidget.AddTreePath(left, "A", "S", "T", "U", "astu")
+	internalwidget.AddTreePath(left, "A", "V", "W", "X", "Y", "Z", "avwxyz")
+
+	l := widget.NewTreeWithStrings(left)
+	l.OnNodeSelected = func(id string) {
+		fmt.Println("LeftTreeNodeSelected:", id)
+	}
+	l.OpenBranch("A")
+	l.OpenBranch("D")
+	l.OpenBranch("E")
+	l.OpenBranch("L")
+	l.OpenBranch("M")
+
+	dir, err := os.UserHomeDir()
+	if err != nil {
+		fyne.LogError("Could not get user home dir", err)
```

Comment:
I think this is missing a fallback, it wil try to list "file://" which is not valid